tests/airportFunctionsUserStory1.spec.js

```diff
@@ -0,0 +1,123 @@
+//USER STORY 1 TESTS
+
```

Comment:
GOOD 5: Each test file is laid out in a fairly neat fashion and each test is formatted in the console in a easy to ready way. The file has relevant comments. These tests files use the AAA method for executing tests.
BAD 5: Each test file imports the other classes, potentially making this tightly coupled. Perhaps adding in mock of each class would have been a more appropriate way to approach this. Some tests are probably missing. For eaxmple, the airport capacity modifier should have tested for negative numbers/made sure that whatever the new number is less than 0 or 1, the current state would allow the airport to have a capacity of negative numbers.
GOALS: Take a step back and list down all the tests. not just ones that a relelvent to context but to the data types you are passing through.If its a number, how can numbers be entered; floats, ints, strings, negatives ,etc. Try and keep my own test files as neat and easy to follow as possible. Try and keep this test files loosely coupled